tests/scripts/check_names.py

```diff
@@ -249,27 +249,39 @@ def comprehensive_parse(self):
             .format(str(self.excluded_files))
         )
 
-        all_macros = self.parse_macros([
+        public_macros = self.parse_macros([
             "include/mbedtls/*.h",
             "include/psa/*.h",
-            "library/*.h",
             "tests/include/test/drivers/*.h",
             "3rdparty/everest/include/everest/everest.h",
             "3rdparty/everest/include/everest/x25519.h"
         ])
-        enum_consts = self.parse_enum_consts([
-            "include/mbedtls/*.h",
+        private_macros = self.parse_macros([
             "library/*.h",
+        ])
+        macros = public_macros + private_macros
```

Comment:
Please keep the name `all_macros`, it's clearer.

Reply:
Ok.